comet/scrapers/base.py

```
@@ -1,12 +1,13 @@
 
-import aiohttp
-
 from comet.scrapers.models import ScrapeRequest
+from comet.utils.network_manager import AsyncClientWrapper
 
 
 class BaseScraper(ABC):
-    def __init__(self, manager, session: aiohttp.ClientSession, url: str = None):
+    impersonate: str | None = None
+
+    def __init__(self, manager, session: AsyncClientWrapper, url: str = None):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)